packages/engine/README.md

```diff
@@ -103,41 +101,28 @@ Depending on your needs, different dependencies are required. Building this proj
         git checkout $latestTag
         ```
 
-- Python [3.7.x] is required for Python initialization or Python behaviors.
-
+- Python [3.10.x] is required for Python initialization and Python behaviors.
   - Python installation guidance from [their website](https://www.python.org/downloads/)
 
 ### macOS Developer Specific Instructions
 
 #### For all macs
 
-Unfortunately, Apple currently doesn't provide a way to resize shared-memory allocations. To work around this allocations need to be sufficiently big such that they will not need to be resized. This can be done by setting the `OS_MEMORY_ALLOC_OVERRIDE` environment variable. A reasonable starting value might be `250000000`, but it is heavily dependent on the memory-requirements of your simulation. This can be done using the command line
+Unfortunately, Apple currently doesn't provide a way to resize shared-memory
+allocations. To work around this, allocations need to be sufficiently big such
+that they will not need to be resized. This can be done by setting the
+`OS_MEMORY_ALLOC_OVERRIDE` environment variable. A reasonable starting value
+might be `250000000`, but it is heavily dependent on the memory-requirements of
+your simulation. This can be done using the command line
```

Comment:
I think we don't want line breaks per paragraph in Markdown (but using soft line breaks instead in IDEs). I don't really mind either way. cc @kachkaev